rosidl_typesupport_opensplice_cpp/bin/rosidl_typesupport_opensplice_cpp

```diff
@@ -59,9 +60,18 @@ def main(argv=sys.argv[1:]):
         return 1
     if rc:
         return rc
+    dds_interface_files = []
+    if not os.path.exists(args.dds_interface_files_file):
+        print("File does not exist: {0}".format(args.dds_interface_files_file), file=sys.stderr)
+        return 1
+    with open(args.dds_interface_files_file, 'r') as f:
+        for line in f.readlines():
+            if not line.strip():
+                continue
+            dds_interface_files.append(line.strip())
```

Comment:
Well if someone would decide to put a file name in there which actually has a leading / trailing space (which is very weird but valid) the "defensive approach would break.